SOOUM/SOOUM/Presentations/Onboarding/NicknameSetting/OnboardingNicknameSettingViewController.swift

```diff
@@ -0,0 +1,221 @@
+        "편집하는", "제작하는", "수리하는", "판매하는",
+        "구매하는", "투자하는", "기획하는", "운영하는",
+        "지원하는", "협력하는", "참여하는", "소통하는",
+        "개선하는", "실천하는", "실험하는", "탐구하는",
+        "수집하는", "배달하는", "전달하는", "연결하는",
+        "조정하는", "선택하는", "결정하는", "준비하는",
+        "확인하는", "수업하는", "연습하는", "발표하는",
+        "기록하는", "정리하는", "대처하는", "해결하는",
+        "조율하는", "탐색하는", "분석하는", "실천하는"
+    ]
+
+    private let nouns = [
+        "강아지", "고양이", "기린", "토끼",
+        "사자", "호랑이", "악어", "코끼리",
+        "판다", "부엉이", "까치", "앵무새",
+        "여우", "오리", "수달", "다람쥐",
+        "펭귄", "참새", "갈매기", "도마뱀",
+        "우산", "책상", "가방", "의자",
+        "시계", "안경", "컵", "접시",
+        "전화기", "자전거", "냉장고", "라디오",
+        "바나나", "케이크", "모자", "열쇠",
+        "지도", "구두", "텀블러", "바구니",
+        "공책", "거울", "청소기", "햄스터",
+        "낙타", "두더지", "돌고래", "문어",
+        "미어캣", "오소리", "다슬기", "해파리",
+        "원숭이", "홍학", "물개", "바다표",
+        "코뿔소", "물소", "개구리", "거북이"
+    ]
+
+    private let maxCount = 8
+
+    private let guideLabelView = OnboardingGuideLabelView().then {
+        $0.titleLabel.text = "반가워요!\n당신을 어떻게 부르면 될까요?"
+        $0.descLabel.text = "닉네임은 추후 변경이 가능해요"
+    }
+    
+    private lazy var nicknameTextField = OnboardingNicknameTextFieldView().then {
+        $0.textField.text = adjectives.randomElement()! + " " + nouns.randomElement()!
+    }
+    
+    private let errorLogStackView = UIStackView().then {
+        $0.axis = .horizontal
+        $0.spacing = 4
+        $0.alignment = .center
+        
+        let imageView = UIImageView().then {
+            $0.image = .error
+            $0.contentMode = .scaleAspectFit
+        }
+        $0.addArrangedSubviews(imageView)
+        $0.isHidden = true
+    }
+    
+    private let errorLogLabel = UILabel().then {
+        $0.typography = .init(
+            fontContainer: BuiltInFont(
+                size: 14,
+                weight: .regular
+            ),
+            lineHeight: 19.6,
+            letterSpacing: 0
+        )
+        // TODO: - 색 수정 필요
+        $0.textColor = .som.red
+        $0.text = "한 글자 이상 입력해주세요"
+    }
+    
+    private let nicknameCountLabel = UILabel().then {
+        $0.typography = .init(
+            fontContainer: BuiltInFont(
+                size: 14,
+                weight: .regular
+            ),
+            lineHeight: 19.6,
+            letterSpacing: 0
+        )
+        // TODO: - 색 수정 필요
+        $0.textColor = .som.gray500
+        $0.text = "1/8"
+    }
+    
+    private let nextButtonView = PrimaryButtonView()
+    
+    // Reactor를 연결하고, 액션과 상태를 바인딩합니다.
+    func bind(reactor: OnboardingNicknameSettingViewReactor) {
+        
+        nicknameTextField.textField.rx.text.orEmpty
+            .observe(on: MainScheduler.instance)
+            .subscribe(onNext: { [weak self] str in
+                guard let self = self else { return }
+                self.nicknameCountLabel.text = "\(str.count)/\(self.maxCount)"
+            })
+            .disposed(by: disposeBag)
```

Comment:
```suggestion
        
        nicknameTextField.textField.rx.text.orEmpty
            .observe(on: MainScheduler.instance)
            .subscribe(with: self) { object, str in
                object.nicknameCountLabel.text = "\(str.count)/\(self.maxCount)"
            }
            .disposed(by: disposeBag)
```
코드 컨벤션을 맞추기 위해 subscribe 내에서 with: self 사용하는 게 좋을 거 같습니다.